.github/workflows/ci_win.yml

```diff
@@ -88,10 +88,30 @@ jobs:
         with:
           name: SYCLomatic_Win_${{ github.sha }}
           path: c2s_obj.7z 
+
       - name: Lit
         run: |
           cd c2s.src/build
           ninja check-clang-c2s -j8
+        shell: bash
+
+      - name: Lit_CUDA8.0
+        run: |
+          cd c2s.src/build
+          set CUDA_PATH="L:/ref/cuda/win/v8.0" && ninja check-clang-c2s -j8
```

Comment:
`set CUDA_PATH="L:/ref/cuda/win/vM.N"` does not seems to work. CUDA 10.2 is still used.

Suggest you remove the quotes and have a try.